albumentations/augmentations/functional.py

```diff
@@ -19,6 +95,19 @@
     np.dtype("float32"): 1.0,
 }
 
+NPDTYPE_TO_OPENCV_DTYPE = {
+    np.uint8: cv2.CV_8U,
+    np.uint16: cv2.CV_16U,
+    np.int32: cv2.CV_32S,
+    np.float32: cv2.CV_32F,
+    np.float64: cv2.CV_64F,
+    np.dtype("uint8"): cv2.CV_8U,
+    np.dtype("uint16"): cv2.CV_16U,
+    np.dtype("int32"): cv2.CV_32S,
+    np.dtype("float32"): cv2.CV_32F,
+    np.dtype("float64"): cv2.CV_64F,
+}
```

Comment:
As we split the code into smaller pieces may be it is better to move these constants (such as `MAX_VALUES_BY_DTYPE` and `NPDTYPE_TO_OPENCV_DTYPE`) to a standalone file?

However, I am also ok with the current approach.